hook.php

```diff
@@ -69,6 +67,8 @@ function plugin_news_install() {
          `plugin_news_alerts_id` INT NOT NULL,
          `itemtype`              VARCHAR(255) NOT NULL,
          `items_id`              INT NOT NULL,
+         `entities_id`           INT NOT NULL,
+         `is_recursive`          TINYINT(1) NOT NULL DEFAULT 1,
```

Comment:
i don't find any migration for this change, did i miss it ?

Reply:
Seems I missed that.